erpnext/controllers/sales_and_purchase_return.py

```diff
@@ -1162,3 +1163,30 @@ def get_available_serial_nos(serial_nos, warehouse):
 def get_payment_data(invoice):
 	payment = frappe.db.get_all("Sales Invoice Payment", {"parent": invoice}, ["mode_of_payment", "amount"])
 	return payment
+
+
+@frappe.whitelist()
+def get_pos_inv_item_returned_qty(invoice, customer, item_name):
+	POS_INVOICE = "POS Invoice"
```

Comment:
We don't use this kind of variable anywhere, you can directly use "POS Invoice" instead of POS_INVOICE